card/mod/05_email/set/all/notify.rb

```diff
@@ -72,21 +72,21 @@ def notable_change?
     Card::Auth.current_id != WagnBotID && followable?
 end
 
-event :notify_followers_after_save,
-      after: :extend, on: :save, when: proc { |ca| ca.notable_change? } do
+event :notify_followers_after_save, :integrate,
+      on: :save, when: proc { |ca| ca.notable_change? } do
```

Comment:
I like the pattern of the line break before the opts (where needed).  seems like styling got much more consistent.